models/stg_hubspot__contact.yml

```diff
@@ -31,26 +31,66 @@ models:
+
+          NOTE: This field is deprecated and will not be populated for connectors utilizing the HubSpot v3 API. This field will be removed in a future release.
       - name: metadata_error
-        description: Any errors that happened the last time the list was processed.
+        description: >
+          Any errors that happened the last time the list was processed.
+
+          NOTE: This field is deprecated and will not be populated for connectors utilizing the HubSpot v3 API. This field will be removed in a future release.
       - name: metadata_last_processing_state_change_at
-        description: A timestamp of the last time that the processing state changed.
+        description: >
+          A timestamp of the last time that the processing state changed.
+
+          NOTE: This field is deprecated and will not be populated for connectors utilizing the HubSpot v3 API. This field will be removed in a future release.
       - name: metadata_last_size_change_at
-        description: A timestamp of the last time that the size of the list changed.
+        description: >
+          A timestamp of the last time that the size of the list changed.
+
+          NOTE: This field is deprecated and will not be populated for connectors utilizing the HubSpot v3 API. This field will be removed in a future release.
       - name: metadata_processing
-        description: |
-              One of DONE, REFRESHING, INITIALIZING, or PROCESSING. 
-              DONE indicates the list has finished processing, any other value indicates that list membership is being evaluated.
+        description: >
+          One of DONE, REFRESHING, INITIALIZING, or PROCESSING. 
+          DONE indicates the list has finished processing, any other value indicates that list membership is being evaluated.
+
+          NOTE: This field is deprecated and will not be populated for connectors utilizing the HubSpot v3 API. This field will be removed in a future release.
       - name: metadata_size
-        description: The approximate number of contacts in the list.
+        description: >
+          The approximate number of contacts in the list.
+
+          NOTE: This field is deprecated and will not be populated for connectors utilizing the HubSpot v3 API. This field will be removed in a future release.
       - name: portal_id
-        description: '{{ doc("portal_id") }}'
+        description: >
+          '{{ doc("portal_id") }}'
+
+          NOTE: This field is deprecated and will not be populated for connectors utilizing the HubSpot v3 API. This field will be removed in a future release.
       - name: updated_timestamp
-        description: A timestamp of the time that the list was last modified.
+        description: >
+          A timestamp of the time that the list was last modified.
+
+          NOTE: This field is deprecated and will not be populated for connectors utilizing the HubSpot v3 API. This field will be removed in a future release.
```

Comment:
Think you can remove this--`updated_timestamp` was moved up the yml file and already exists, and this field isn't being deprecated.